tests/methods/integrals-1/input.dat

```diff
@@ -0,0 +1,34 @@
+#! Generated using commit GITCOMMIT
+
+import forte
+
+refscf = -112.67064846447627
+reffci = -112.74446815362198
+
+molecule {
+0 1
+C
+O 1 R
+R = 2.5
+units bohr
+}
+
+set {
+  basis cc-pVDZ
+  scf_type pk
+  e_convergence 14
+}
+
+set forte {
+  job_type fci
+  frozen_docc      [2 ,0 ,0 ,0]
+  restricted_docc  [2 ,0 ,0 ,0]
+  active           [2 ,2 ,2 ,2]
+  e_convergence 12
+}
+
+energy('scf')
+compare_values(refscf, get_variable("CURRENT ENERGY"),10, "SCF energy") #TEST
+
```

Comment:
Minor thing here. If the reference for forte is not set, a scf will run automatically. Thus, there is no need to run `energy('scf')`. Or it should be the following
```
Escf, wfn = energy('scf', return_wfn=True)
energy('forte', ref_wfn=wfn)
```
Since the documentation is limited, we should make our test cases the best guide.